components/ui/AppCard.vue

```diff
@@ -37,11 +37,9 @@
               class="app-card__custom-pill"
             >
               {{ tag.label }}
-              <bx-tooltip direction="bottom">
-                <bx-tooltip-body>
-                  {{ tag.description }}
-                </bx-tooltip-body>
-              </bx-tooltip>
+              <bx-tooltip-icon :body-text="tag.description" direction="bottom">
```

Comment:
Same as before, I think the use of the default tooltip component is more appropriate here